HellBrick.NoCapture/Analyzer/NoCaptureAnalyzer.cs

```diff
@@ -65,6 +65,31 @@ Diagnostic diagnostic
 
 					bool HasNoCaptureAttribute( ISymbol symbol )
 						=> symbol.GetAttributes().Any( a => a.AttributeClass.Name == _noCaptureAttributeName );
+
+					IEnumerable<ISymbol> GetCapturedSymbols()
+					{
+						DataFlowAnalysis dataFlow = nodeContext.SemanticModel.AnalyzeDataFlow( nodeContext.Node );
+						ImmutableArray<ISymbol> capturedSymbols = dataFlow.Captured.IntersectWith( dataFlow.ReadInside );
+
+						if ( capturedSymbols.IsEmpty )
+							return Enumerable.Empty<ISymbol>();
+
+						IEnumerable<ISymbol> childDeclared = nodeContext
+							.Node
+							.DescendantNodes( descendIntoChildren: _ => true, descendIntoTrivia: false )
+							.Prepend( nodeContext.Node )
```

Comment:
I think simple `DescendantNodesAndSelf( descendIntoChildren: _ => true, descendIntoTrivia: false )` should do the trick.

Reply:
Whoops, missed this one. Done.